sensitivity-analysis-api/src/test/resources/factorRef.json

```diff
@@ -1,5 +1,5 @@
 {
-  "functionType" : "BRANCH_ACTIVE_POWER",
```

Comment:
I don't understand very well why we have this change.

Reply:
This is used in  a json test. I replaced a deprecated function type by a non deprecated one for the serialization test. See SensitivityAnalysisParameterTest.testJson.

Reply:
I was wondering about the "2" indeed, but ok.